label_options/example-program-label-options.json

```diff
@@ -84,8 +88,9 @@
     },
     "es": {
       "walk": "Caminando",
-      "bike": "Bicicleta",
       "e-bike": "e-bicicleta",
+      "bike": "Bicicleta",
+      "bikeshare": "Bicicleta compartida",
```

Comment:
It looks like you were initially missing `bikeshare` from the Spanish translation. This might happen again, particularly for actual partner-specified files. I am not 100% sure what will happen if/when we have a missing translation, but I don't think it will be good. 

I think we should write a small script to validate the format of the specified file (if any) is internally consistent, and run it using GitHub actions on every PR.

@JGreenlee @Abby-Wheelis can one of you file an issue for this? We can figure out who should work on it.

Reply:
If one translation was missing, the mode/purpose would just show up blank.
If the entire language was missing from here, it would probably throw an error.

Reply:
I actually made a tool last week to help us find i18n discrepancies between `e-mission-phone` and `e-mission-translate`
https://codepen.io/jackofhearts/full/YzdyZMR

Although, the script you are talking about sounds like it would be specifically for validating config files. We could validate label options config as well as the dynamic config as a whole

Reply:
And if we someday make a tool to build dynamic configs, it will be helpful to have that validation logic anyway

Reply:
Yeah! Let's file an issue for the validation and add in a GitHub action to test it.

In this repo, for every pull request, we want to validate:
- dynamic config in general
- label options config

In the translate repo, for every pull request, we want to validate:
- that all the keys are in all files

This will also get y'all used to writing automated tests

> And if we someday make a tool to build dynamic configs, it will be helpful to have that validation logic anyway

I was actually thinking of a form-like tool to build dynamic configs, so it should generate valid configs anyway. But people might tweak configs without going through the tool - it is always better to validate before merging...